python/pdstools/plots/plot_base.py

```diff
@@ -482,7 +482,7 @@ def plotOverTime(
                 df.groupby_dynamic("SnapshotTime", every=every, by=groupby)
                 .agg(
                     [
-                        (pl.sum("Positives") / pl.sum("ResponseCount")).alias(
+                        weighed_average_polars("SuccessRate", "ResponseCount").alias(
```

Comment:
Why would we want to use weighed average success rate? I don't think that's right, is it?